nwb_conversion_tools/utils/si013nwbephyswriter.py

```diff
@@ -126,10 +126,9 @@ def _get_num_frames(self, segment_index=0):
 
     @default_return([])
     def _get_recording_times(self, segment_index=0):
-        if self.recording is not None:
-            if self.recording._times is None:
-                return np.arange(0, self._get_num_frames()*self._get_sampling_frequency(), self._get_sampling_frequency())
-            return self.recording._times
+        if self.recording._times is None:
```

Comment:
even with the decorator, if `self.recording is None` this will fail because it would try to get `self.recording._times`

Reply:
the decorator also checks if we're calling some method/attribute on a NoneType (if self.recording/sorting etc are None). Then it also returns the default value also.